packages/allure-js-commons/src/AllureCommandStep.ts

```diff
@@ -57,6 +57,10 @@ export interface AllureCommandStep<T = MetadataMessage> {
     content: Buffer | string,
     options: ContentType | string,
   ): void | Promise<void>;
+
+  description(content: string): void | Promise<void>;
+
+  descriptionHtml(content: string): void | Promise<void>;
```

Comment:
`descriptionHtml` is very technical and has limitations. For regular users, we suggest always using `description` instead. Maybe only expose `description` for now?

Reply:
Ok, no problem

Reply:
👍